src/components/TitleForm.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019-2020 @Premiurly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import styled from '@xstyled/styled-components';
+import { Form } from 'antd';
+import React from 'react';
+
+import messages from '../util/messages';
+
+interface Props {
+	className?: string
+	errorTitle?: any
+	onChange?: any
+	value?: string
+}
+
+const TitleForm = ({ className, errorTitle, onChange, value = '' }:Props): JSX.Element => {
+
+	return (
+		<div className={className}>
+			<Form>
+				<Form.Item >
+					<label>Title</label>
+					<input
```

Comment:
Please use the ant-design Input, it should handle all the error messages on it's own.